### PR TITLE
Fix for service catalog rest test and requests

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/services/test_service_catalogs.py -k "test_order_catalog_item_via_rest" -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f0f3b4b6c7e3f919d5ef641076dae1dac2422ec2:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/services/requests.py [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 buttons = Region(
     locators=dict(
-        approve="//*[@title='Approve this Request']/img",
```
Comment (psav) [inline cfme/services/requests.py]: This is the same change for 5.5 and 5.6?

File: cfme/tests/services/test_service_catalogs.py [hunk 2]
```diff
@@ -69,6 +69,8 @@ def test_order_catalog_item_via_rest(
     assert len(template) == 1
     template, = template
     req = template.action.order()
+    cells = {'Description': catalog_item.name}
+    requests.approve_request(cells, "Approved")
```
Comment (psav) [inline cfme/tests/services/test_service_catalogs.py]: Should this be an assertion?

File: cfme/services/requests.py (hunk 1 above)
Comment (lehinevych) [inline cfme/services/requests.py]: Yes

File: cfme/tests/services/test_service_catalogs.py (hunk 2 above)
Comment (lehinevych) [inline cfme/tests/services/test_service_catalogs.py]: It is a bug 1347409, I'm going to add marker

**Comment by dajoRH** [comment]
Lint report for commit 69e8465e721526a321fa69df8f435942af3bdf70:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**